send-report.sh

```diff
@@ -28,8 +28,8 @@ Content-Id: <$2>" >> "$file_upload"
 echo "<html>
 <body>
     <div>
-        <p>Please see the screenshot included below</p>
-        <p>cloud.gov team</p>
+        <p>Please see the screenshot included below. Convert timestamps using a tool like <a href="https://www.epochconverter.com/">epochconverter.com</a>></p>
```

Comment:
@wslack Since this HTML is being included with double quotes (`"`) via the `echo` in line 28, you'll either need to use single quotes (`'`) in the `href`, or escape with a backslash (`\"`). Also, you have an extra `>` at the end.